packages/NodeTypeResolver/Reflection/BetterReflection/RectorBetterReflectionSourceLocatorFactory.php

```diff
@@ -20,14 +21,14 @@ public function __construct(
     ) {
     }
 
-    public function create(): MemoizingSourceLocator
+    public function create(): SourceLocator
     {
         $phpStanSourceLocator = $this->betterReflectionSourceLocatorFactory->create();
 
         // make PHPStan first source locator, so we avoid parsing every single file - huge performance hit!
-        $aggregateSourceLocator = new AggregateSourceLocator([$phpStanSourceLocator, $this->intermediateSourceLocator]);
+        return new AggregateSourceLocator([$phpStanSourceLocator, $this->intermediateSourceLocator]);
 
-        // important for cache
-        return new MemoizingSourceLocator($aggregateSourceLocator);
+        // important for cache, but should rebuild for tests
+        //        return new MemoizingSourceLocator($aggregateSourceLocator);
```

Comment:
comment should live belong to Stmt, so should be placed before `return`

Reply:
This will be revert, I think issue was elsewhere. Thanks for pointing out :+1: